rusk-wallet/CHANGELOG.md

```diff
@@ -25,6 +25,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix contract init parsing [#3602]
 - Fix contract call non-interactive parsing [#3602]
 
+### Added
+
+- Add `clippy::padantic`
```

Comment:
Instead of mention the clippy, I would state the changes in public methods (removal of async or not returning result anymore)

Reply:
You are right, I'll change it